package.json

```diff
@@ -14,7 +14,8 @@
     "lint-code": "tsdx lint components pages util --quiet",
     "lint-docs": "documentation lint {components,pages,util}/**/*.js",
     "lint-types": "yarn tsc",
-    "predev": "yarn"
+    "predev": "yarn",
+    "test": "cp -r __tests__/e2e/env.test .env.build && yarn jest __tests__/e2e/"
```

Comment:
Include e2e in the script name (and propagate to the yaml scripts)

Reply:
Added in ee7126aa0a34c6d7926d6496ed753c56e3098e76!